test/path.jl

```diff
@@ -0,0 +1,5 @@
+@test @unix_only expanduser("~")[1] != ENV["HOME"]
```

Comment:
this should be `@unix_only` `@test` (though not sure whether the `@unix_only` is needed)?